### ISSUE TITLE
Sort

### ISSUE THREAD

**Issue by rawsh** [posativ/isso#440]
Title

**Comment by blatinier** [comment]
Is this an error? Should I close? If not please provide more information. Sort what, on what, where…

**Comment by rawsh** [comment]
How can we sort comments by new, votes, old, etc.

**Comment by blatinier** [comment]
Nothing available so far to do it.
It's mentioned here: #4  and there's a PR (which fails at implementing it completly) here: #322

**Comment by rawsh** [comment]
@blatinier client side:

```
#isso-root {
    display: flex;
    flex-direction: column-reverse;
}
.isso-comment {
    flex: 0;
    width: 100%;
}
```

**Comment by Jinxiapu** [comment]
It may be useful and easy to add a url parma to [fetch api](https://github.com/posativ/isso/blob/dfbc80d5b645f0754de6884078db29ac6a6ad642/isso/views/comments.py#L732), because there are some parma(asc, order_by) in [db fetch api](https://github.com/posativ/isso/blob/dfbc80d5b645f0754de6884078db29ac6a6ad642/isso/db/comments.py#L199) for the order of comments.

**Comment by rawsh** [comment]
Yeah that would be a clean solution. It could be a client option then in the js

**Comment by ix5** [comment]
@budui @rawsh do you feel like working on this?
